src/test/java/uk/gov/hmcts/cmc/claimstore/rpa/RoboticsNotificationServiceTest.java

```diff
@@ -0,0 +1,222 @@
+import uk.gov.hmcts.cmc.domain.exceptions.BadRequestException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaim;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleCountyCourtJudgment;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleResponse;
+
+import java.time.LocalDate;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class RoboticsNotificationServiceTest {
+
+    @InjectMocks
+    RoboticsNotificationServiceImpl roboticsNotificationService;
+    @Mock
+    private UserService userService;
+    @Mock
+    private ClaimService claimService;
+    @Mock
+    private MoreTimeRequestedNotificationService moreTimeRequestedNotificationService;
+    @Mock
+    private DefenceResponseNotificationService defenceResponseNotificationService;
+    @Mock
+    private RequestForJudgementNotificationService ccjNotificationService;
+    @Mock
+    private ResponseDeadlineCalculator responseDeadlineCalculator;
+    @Mock
+    private AppInsightsExceptionLogger appInsightsExceptionLogger;
+    @Mock
+    private DocumentGenerator documentGenerator;
+    @Mock
+    private PaidInFullNotificationService paidInFullNotificationService;
+
+    private static final String FORE_NAME = "forename";
+    private static final String SURNAME = "surname";
+    private static final UserDetails USER_DETAILS = SampleUserDetails.builder()
+        .withForename(FORE_NAME)
+        .withSurname(SURNAME)
+        .build();
+    private static final String REFERENCE_NUMBER = "000MC001";
+    private static final GeneratePinResponse PIN_RESPONSE = new GeneratePinResponse("pin", "userid");
+    private static final String RPA_STATE_SUCCEEDED = "succeeded";
+    private static final String RPA_STATE_FAILED = "failed: reason";
+
+    @Before
+    public void setup() {
+        userService = mock(UserService.class);
+        claimService = mock(ClaimService.class);
+        moreTimeRequestedNotificationService = mock(MoreTimeRequestedNotificationService.class);
+        ccjNotificationService = mock(RequestForJudgementNotificationService.class);
+        responseDeadlineCalculator = mock(ResponseDeadlineCalculator.class);
+        appInsightsExceptionLogger = mock(AppInsightsExceptionLogger.class);
+        documentGenerator = mock(DocumentGenerator.class);
+        defenceResponseNotificationService = mock(DefenceResponseNotificationService.class);
+        paidInFullNotificationService = mock(PaidInFullNotificationService.class);
```

Comment:
If you run with the Mockito JUnit runner (or extend with the mockito extension in JUnit 5) and annotate the fields you don't need to explicitly instantiate them